packages/x-scheduler/src/timeline/Timeline.test.tsx

```diff
@@ -1,44 +1,43 @@
 import { screen } from '@mui/internal-test-utils';
 import { diffIn } from '@mui/x-scheduler-headless/use-adapter';
 import { Timeline } from '@mui/x-scheduler/timeline';
+import {
+  adapter,
+  createSchedulerRenderer,
+  DEFAULT_TESTING_VISIBLE_DATE,
+  DEFAULT_TESTING_VISIBLE_DATE_STR,
+  EventBuilder,
+} from 'test/utils/scheduler';
 import { SchedulerEvent, CalendarResource, TimelineView } from '@mui/x-scheduler-headless/models';
-import { adapter, createSchedulerRenderer } from 'test/utils/scheduler';
 
 const baseResources: CalendarResource[] = [
   { id: 'resource-1', title: 'Engineering', eventColor: 'blue' },
   { id: 'resource-2', title: 'Design', eventColor: 'jade' },
 ];
 
 const baseEvents: SchedulerEvent[] = [
-  {
-    id: 'event-1',
-    title: 'Spec Review',
-    start: adapter.date('2025-07-03T09:00:00Z'),
-    end: adapter.date('2025-07-03T10:00:00Z'),
-    resource: 'resource-1',
-  },
-  {
-    id: 'event-2',
-    title: 'UX Sync',
-    start: adapter.date('2025-07-03T11:00:00Z'),
-    end: adapter.date('2025-07-06T12:00:00Z'),
-    resource: 'resource-2',
-  },
-  {
-    id: 'event-3',
-    title: 'Architecture Session',
-    start: adapter.date('2025-07-04T13:00:00Z'),
-    end: adapter.date('2025-08-04T14:30:00Z'),
-    resource: 'resource-1',
-  },
+  EventBuilder.new()
+    .title('Spec Review')
```

Comment:
Which DX do you think is the best for the tests?

1. Explicit title:

  ```ts
  const event = EventBuilder.new().title('My title').build();
  
  const element = screen.getByText('My title');
  ```
  
2. Implicit title

  ```ts
  const event = EventBuilder.new().build()
  
  const element = screen.getByText(event.title);
  ```